client/scss/settings/_variables.scss

```
@@ -59,8 +60,8 @@ $color-header-bg: $color-teal;
 
 $color-fieldset-hover: $color-grey-5;
 $color-input-border: $color-grey-4;
-$color-input-focus: lighten(desaturate($color-teal, 40), 72);
-$color-input-focus-border: lighten(saturate($color-teal, 12), 10);
+$color-input-focus: var(--color-input-focus);
+$color-input-focus-border: var(--color-input-focus);
```

Looks like this should be:

```suggestion
$color-input-focus-border: var(--color-input-focus-border);
```